src/modules/universe/actions/sync-universe.js

```diff
@@ -16,7 +16,7 @@ const syncUniverse = (callback = logError) => (dispatch, getState) => {
     const isForking = forkingMarket !== '0x0000000000000000000000000000000000000000'
     if (isForking) {
       // todo: use new load market data if not loaded when it's available
-      dispatch(loadMarketsInfo([forkingMarket]))
+      dispatch(loadMarketsInfoIfNotLoaded([forkingMarket]))
```

Comment:
@bthaile Should we replace `loadMarketsInfo` with `loadMarketsInfoIfNotLoaded` everywhere?

Reply:
No, some places we need to force a re-loading because reporting state has
changes